### PR TITLE
Prepare for release 4.0.2-KT15

### LINKED ISSUE #263: [BUG] Crash in Store 4.0 with Kotlin 1.5.0-RC
**Describe the bug**
Application crash after update Kotlin from 1.4.32 to 1.5.0-RC, no other changes in code.

Catch Exception:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ru.superapplication.android.app.debug, PID: 18674
    java.lang.NoSuchMethodError: No static method getHours(I)D in class Lkotlin/time/DurationKt; or its super classes (declaration of 'kotlin.time.DurationKt' appears in /data/app/~~l_eMNOPYSnv8XUlDP7ycgA==/ru.superapplication.android.app.debug-ziiU-phO368T6tA7R1CseQ==/base.apk!classes33.dex)
        at com.dropbox.android.external.store4.StoreDefaults.<clinit>(StoreDefaults.kt:15)
        at com.dropbox.android.external.store4.RealStoreBuilder.<init>(StoreBuilder.kt:91)
        at com.dropbox.android.external.store4.StoreBuilder$Companion.from(StoreBuilder.kt:76)
```

### PR DESCRIPTION
**Pull request by digitalbuddha**
Preparing for release of Kotlin 1.5 compat version.
Closes #263

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,7 +52,9 @@ allprojects {
 ext {
     // POM file
     GROUP = "com.dropbox.mobile.store"
-    VERSION_NAME = "4.0.2-KT15-SNAPSHOT"
+    //e2c5cbfb31bbaa9793f430a25c750195c414ac44 was the last commit compatible with kotlin 1.4
+    //if a non kotlin 1.5 release is needed, branch off that commit and cherry pick any newer changes
```
Comment (mandybess) [inline build.gradle:56]: non or < ?

**Review by mandybess — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (digitalbuddha) [inline build.gradle:56]:
```suggestion
    //if a release with lower compatiblity than kotlin 1.5 is needed, branch off the above commit and cherry pick any newer changes
```

Comment (digitalbuddha) [reply build.gradle:56]: @mandybess better or worse? (See comment above)

**Review by dehboxturtle-dropbox — DISMISSED** [review]
LGTM pending comment from mandybess

File: build.gradle (hunk 1 above)
Comment (mandybess) [reply build.gradle:56]: better imo 👍 thanks!

**Comment by digitalbuddha** [comment]
updated wording

**Review by chris-mitchell — APPROVED** [review]

**Pull request merged**